backend/de.metas.edi/src/main/java/de/metas/edi/api/impl/DesadvBL.java

```diff
@@ -1125,6 +1138,18 @@ public ImmutableList<ITranslatableString> createMsgsForDesadvsBelowMinimumFulfil
 		return result.build();
 	}
 
+	@Override
+	public void updateQtyOrdered_OverrideFromShipSchedAndSave(@NonNull final I_M_ShipmentSchedule schedule)
+	{
+		final I_C_OrderLine orderLineRecord = InterfaceWrapperHelper.create(schedule.getC_OrderLine(), I_C_OrderLine.class);
```

Comment:
pls avoid `schedule.getC_OrderLine()`.
use the right Order DAO API to get the order line.